standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/messaging/event/filters/ReplEventFilter.java

```diff
@@ -33,7 +33,7 @@ public ReplEventFilter(final ReplScope replScope) {
 
   // Return false if all the tables are included, as bootstrap during rename is done only in case filter set for tables.
   boolean isAlterDuringTableLevelReplication(final NotificationEvent event) {
-    if (replScope.includeAllTables() || !replScope.getDbNamePattern().matcher(event.getDbName()).matches()) {
+    if (!replScope.dbIncludedInReplScope(event.getDbName())) {
       return false;
     }
     return event.getEventType().equals(MessageBuilder.ALTER_TABLE_EVENT);
```

Comment:
Can combine to one line as replScope.dbIncludedInReplScope() && event.getEventType().equals().